src/os_posix/os_fs.c

```diff
@@ -748,7 +748,10 @@ __posix_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session, const cha
         f |= O_CLOEXEC;
 #endif
         WT_SYSCALL_RETRY(((pfh->fd = open(name, f, 0444)) == -1 ? -1 : 0), ret);
-        if (ret != 0)
+        /* Return error if the file not found during rollback to stable. */
+        if (F_ISSET(session, WT_SESSION_ROLLBACK_TO_STABLE) && ret != 0)
```

Comment:
Looks like it was already returning an error. Do we expect this to happen in the rollback to stable case now?

Reply:
This error `ENOENT` is returned so that RTS can ignore the corrupted tables [here](https://github.com/wiredtiger/wiredtiger/blob/develop/src/txn/txn_rollback_to_stable.c#L1251) and proceed with other tables. These corruptions will get fixed as part of the salvage operation.